salt/modules/openscap.py

```diff
@@ -155,10 +155,11 @@ def xccdf_eval(
         proc = subprocess.Popen(
             cmd_opts, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=tempdir
         )
-        (stdoutdata, error) = proc.communicate()
+        (_, error) = proc.communicate()
+        error = error.decode("ascii", errors="ignore")
```

Comment:
If Salt is supposed to treat all output as text, and non binary output, then why not fix the subprocess call?
```python
ret = subprocess.run(cmd_opts, capture_output=True, text=True)
error = ret.stderr
```
etc...